### PR TITLE
add student image to gallery section

### PR DESCRIPTION
**Pull request by lazypanda10117**

### CODE REVIEW

**Comment by prwatch** [comment]
DEMO: http://lazypanda10117.github.io/gci15.fossasia.org

File: _data/gallery.yml [hunk 1]
```diff
@@ -219,9 +219,15 @@
     author: Baidyanth Kundu
 
   -
+<<<<<<< HEAD
```
Comment (mananwason) [inline _data/gallery.yml]: Remove this please. This is used for indicating conflicts by git.

File: _data/gallery.yml [hunk 2]
```diff
@@ -219,9 +219,15 @@
...
+    name: PHP App: Hello [name]
+    image: php hello [name] screenshot.png
+    author: Jeffrey Kam
+=======
```
Comment (mananwason) [inline _data/gallery.yml]: This too!!

File: _data/gallery.yml [hunk 3]
```diff
@@ -219,9 +219,15 @@
...
     name: JAVA Quiz
     image: JQ.PNG
     author: Baidyanath Kundu
+>>>>>>> c5606ed64e64b3f97c97efa1017f53eed183f66f
```
Comment (mananwason) [inline _data/gallery.yml]: and this!

File: index.html [hunk 4]
```diff
@@ -524,6 +524,14 @@ <h2 class="photostack-title"><a href="https://github.com/9at8/Distance-Finder" t
             </figcaption>
           </figure>
           <figure>
+<<<<<<< HEAD
```
Comment (mananwason) [inline index.html]: again

File: index.html [hunk 5]
```diff
@@ -524,6 +524,14 @@ <h2 class="photostack-title"><a href="https://github.com/9at8/Distance-Finder" t
...
+              <img src="img/gallery/php hello [name] screenshot.png" alt="img05" height="240" width="240">
+              <figcaption>
+                <h2 class="photostack-title"><a href="https://github.com/lazypanda10117/Hello-Jeffrey-Kam-PHP" target="_blank">PHP App: Hello [name]</a></h2>
+                <h3>Jeffrey Kam</h3>
+              </figcaption>
+            </figure>
+=======
```
Comment (mananwason) [inline index.html]: +1

File: index.html [hunk 6]
```diff
@@ -537,6 +545,7 @@ <h2 class="photostack-title"><a href="http://anshuman73.hol.es/FileMetadata/" ta
               <h3>Anshuman Agarwal</h3>
             </figcaption>
           </figure>
+>>>>>>> c5606ed64e64b3f97c97efa1017f53eed183f66f
```
Comment (mananwason) [inline index.html]: +1

File: index.html [hunk 7]
```diff
@@ -298,17 +298,17 @@ <h2 class="photostack-title">Open Source Collage</h2>
               </figcaption>
             </figure>
             <figure>
-              <img src="img/gallery/apache.png" alt="img03" height="240" width="240">
+              <img src="img/gallery/personalpage.png" alt="img05" height="240" width="240">
               <figcaption>
-                <h2 class="photostack-title"><a href="https://github.com/Yathannsh/PHP_APP" target="_blank" target="_blank" >Configuring Apache with PHP</a></h2>
-                <h3>Yathannsh Kulshrestha</h3>
```
Comment (mananwason) [inline index.html:304]: Why are you deleting this?

File: index.html [hunk 8]
```diff
@@ -298,17 +298,17 @@ <h2 class="photostack-title">Open Source Collage</h2>
...
+                <h2 class="photostack-title"><a href="http://github.com/vldfr/Profile_page/" target="_blank">Personal Page</a></h2>
+                <h3>Vlad Fara</h3>
               </figcaption>
             </figure>
             <figure>
-              <img src="img/gallery/personalpage.png" alt="img05" height="240" width="240">
```
Comment (mananwason) [inline index.html:308]: Why changing this?

File: index.html [hunk 9]
```diff
@@ -298,17 +298,17 @@ <h2 class="photostack-title">Open Source Collage</h2>
...
+              <img src="img/gallery/apache.png" alt="img03" height="240" width="240">
               <figcaption>
-                <h2 class="photostack-title"><a href="http://github.com/vldfr/Profile_page/" target="_blank">Personal Page</a></h2>
-                <h3>Vlad Fara</h3>
```
Comment (mananwason) [inline index.html:311]: Why are you changing this?

**Comment by mananwason** [comment]
@lazypanda10117 Please see my comments

**Comment by lazypanda10117** [comment]
Thank you for the comments. I think I've fixed the problem above.

**Pull request closed without merging**